### PR TITLE
WooCommerce Orders: Update the Add Order button

### PR DESCRIPTION
**Pull request by ryelle**
This PR removes the OrderHeader component- the single order & orders list view have different header actions, so it makes more sense to call ActionHeader directly. It also updated the Add Order link to go to wp-admin, since adding orders is not included in v1.

To test

1. View the orders list: `/store/orders/:site`
2. Clicking Add Order should take you to wp-admin
3. View a single order: `/store/order/:site/:orderId`
4. The action should now say Save Order, and clicking it does nothing (yet)

### CODE REVIEW

**Comment by justinshreve** [comment]
@kellychoffman can confirm, but I think we just want to completely remove the `Add Order` button for now (and add the ability to add orders post v1), rather than redirect them to `wp-admin` for it.

File: client/extensions/woocommerce/app/order/index.js [hunk 1]
```diff
@@ -46,7 +49,9 @@ class Order extends Component {
 		];
 		return (
 			<Main className={ className }>
-				<OrderHeader siteSlug={ site.slug } breadcrumbs={ breadcrumbs } />
+				<ActionHeader breadcrumbs={ breadcrumbs }>
+					<Button primary onClick={ this.saveOrder }>{ translate( 'Save Order' ) }</Button>
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/index.js]: Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Save New Order' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woo-orders-action-header)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/orders/index.js [hunk 2]
```diff
@@ -8,17 +8,21 @@ import React, { Component } from 'react';
...
 import OrdersList from './orders-list';
 
 class Orders extends Component {
 	render() {
-		const { className, site, translate } = this.props;
+		const { className, createOrderLink, translate } = this.props;
 		return (
 			<Main className={ className }>
-				<OrderHeader siteSlug={ site ? site.slug : false } breadcrumbs={ ( <span>{ translate( 'Orders' ) }</span> ) } />
+				<ActionHeader breadcrumbs={ ( <span>{ translate( 'Orders' ) }</span> ) }>
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/orders/index.js]: :disappointed: 99 existing translations will be lost with this change.
If the string must change, the following option has already been translated **87** times:
`translate( 'Order:' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by kellychoffman** [comment]
> @kellychoffman can confirm, but I think we just want to completely remove the Add Order button for now (and add the ability to add orders post v1), rather than redirect them to wp-admin for it.

Yes, let's remove it until we have the ability to edit orders.

File: client/extensions/woocommerce/app/orders/index.js [hunk 3]
```diff
@@ -8,17 +8,17 @@ import React, { Component } from 'react';
...
 import OrdersList from './orders-list';
 
 class Orders extends Component {
 	render() {
-		const { className, site, translate } = this.props;
+		const { className, translate } = this.props;
 		return (
 			<Main className={ className }>
-				<OrderHeader siteSlug={ site ? site.slug : false } breadcrumbs={ ( <span>{ translate( 'Orders' ) }</span> ) } />
+				<ActionHeader breadcrumbs={ ( <span>{ translate( 'Orders' ) }</span> ) } />
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/orders/index.js]: :disappointed: 99 existing translations will be lost with this change.
If the string must change, the following option has already been translated **87** times:
`translate( 'Order:' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/order/index.js [hunk 4]
```diff
@@ -46,10 +66,12 @@ class Order extends Component {
 		];
 		return (
 			<Main className={ className }>
-				<OrderHeader siteSlug={ site.slug } breadcrumbs={ breadcrumbs } />
+				<ActionHeader breadcrumbs={ breadcrumbs }>
+					<Button primary onClick={ this.saveOrder } disabled={ isSaving }>{ translate( 'Save Order' ) }</Button>
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/index.js]: Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Save New Order' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woo-orders-action-header)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/order/order-details.js [hunk 5]
```diff
@@ -1,36 +1,87 @@
...
+			name: translate( 'On Hold' ),
+		}, {
+			value: 'completed',
+			name: translate( 'Completed' ),
+		}, {
+			value: 'cancelled',
+			name: translate( 'Cancelled' ),
+		}, {
+			value: 'refunded',
+			name: translate( 'Refunded' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-details.js]: Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Refunded To' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woo-orders-action-header)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/extensions/woocommerce/app/order/order-details.js [hunk 6]
```diff
@@ -1,36 +1,87 @@
...
+			name: translate( 'Pending payment' ),
+		}, {
+			value: 'processing',
+			name: translate( 'Processing' ),
+		}, {
+			value: 'on-hold',
+			name: translate( 'On Hold' ),
+		}, {
+			value: 'completed',
+			name: translate( 'Completed' ),
```
Comment (a8ci18n) [inline client/extensions/woocommerce/app/order/order-details.js]: Hi! I've found a possible matching string that has already been translated **52** times:
`translate( '&laquo; completed' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woo-orders-action-header)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Pull request merged**